frontend/app/src/components/settings/data-security/DataManagement.vue

```diff
@@ -52,19 +48,18 @@ import {
 } from '@/services/session/consts';
 import { Purgeable } from '@/services/session/types';
 import { ACTION_PURGE_CACHED_DATA } from '@/store/session/const';
-import { SuccessMessage } from '@/store/types';
 import { SUPPORTED_EXCHANGES, SupportedExchange } from '@/types/exchanges';
 import { Module } from '@/types/modules';
 
 @Component({
-  components: { PurgeSelector, StatusButton, ConfirmDialog },
+  components: { SettingCategory, PurgeSelector, StatusButton, ConfirmDialog },
```

Comment:
Is it too much effort to also re-write this component in composition?

Reply:
I certainly missed this.